style.css

```diff
@@ -404,3 +404,124 @@ nav #mylogo {
   justify-content: space-between;
   list-style: none;
 }
+
+.container {
+  display: flex;
+  width: 100%;
+  height: auto;
+  flex-direction: column;
+  border-top-left-radius: 80px;
+  padding: 115px 25px;
+  background-color: #6070ff;
+  padding-bottom: 40px;
+  position: relative;
+}
+
+.container::before {
+  content: "";
+  display: block;
+  width: 100%;
+  height: 9rem;
+  background-image: url("./images/icon01.png"), url("./images/icon02.png");
+  background-size: 6rem, 2rem;
+  background-repeat: no-repeat;
+  background-position: 100% 99%, 21% 52%;
+  position: absolute;
+  right: 0%;
+  top: 1%;
+}
+
+.container::after {
+  content: "";
+  display: block;
+  width: 19rem;
+  height: 9rem;
+  background-image: url("./images/icon03.png");
+  background-size: 5rem;
+  background-repeat: no-repeat;
+  background-position: 100% 99%;
+  position: absolute;
+  right: 15%;
+  bottom: 9%;
+}
```

Comment:
- [ ] Good job implementing the suggested changes by the previous reviewer but there is another issue.
these lines of code that add the decoration images are somehow blocking your form submit button to function properly. kindly either consider removing them and then consider downloading the whole background decoration image from the figma and then placing it in the form section as a background image, or you can adjust it as it is now and make sure that your form submission button is working.

Reply:
# He can also add the property **z-index: 1** to the children components     :grin:

z-index: 0 is the default value of all the components.
The pseudo class is blocking the children components because of also be at the same level of z-index.
So in order to avoid playing to much with the with, height and background-size to avoid that issue. It is easier to give all the chindren components of container a z-index: 1 so they would not be blocked.

    .container > * {
      z-index: 1;
    }

    .container > * {
      z-index: 1;
    }